sound/soc/sof/ipc4-topology.h

```diff
@@ -148,13 +148,17 @@ struct ipc4_pipeline_set_state_data {
  * struct sof_ipc4_available_audio_format - Available audio formats
  * @base_config: Available base config
  * @out_audio_fmt: Available output audio format
+ * @input_audio_fmts: Available input audio formats
  * @ref_audio_fmt: Reference audio format to match runtime audio format
  * @dma_buffer_size: Available Gateway DMA buffer size (in bytes)
- * @audio_fmt_num: Number of available audio formats
+ * @audio_fmt_num: Number of available audio formats. Audio formats defined in topology always
+ * contain both the input and output formats for a particular pin. So audio_fmt_num is the same
```

Comment:
I still don't get the wording used "contain both the input and output formats for a particular pin", the two parts of that wording seem to contradict each other.

Reply:
in that case don't touch change the kernel-doc description if this is removed later